website/docs/targets.md

```diff
@@ -2,12 +2,32 @@
 title: Targets
 ---
 
-This section provides an overview of the targets that Dalec supports. At this time, supported targets are `mariner2`, `azlinux3`, and `windowscross`.
-
-Many components, such as package dependencies and base images, are specific to a distro or a subset of distros. The dalec spec allows you to move these distro specific things into a `target`.
+DALEC is designed to support building artifacts for a number of different
+systems.
+DALEC refers to these in the [spec](spec.md) as "targets".
+When executing a build with Docker these targets can be specified with the
+`--target=<target>` flag.
 
 ## Available Targets
 
+DALEC includes a number of built-in targets that you can either use in your spec.
```

Comment:
oh wow, I was just referencing this too. Thanks for adding more targets to the list!